.hashibot.hcl

```
@@ -826,6 +827,8 @@ behavior "pull_request_path_labeler" "service_labels" {
       "aws/internal/service/cloudwatchlogs/**/*",
       "**/*_cloudwatch_log_*",
       "**/cloudwatch_log_*"
```

Oops forgot the extra comma here

```suggestion
      "**/cloudwatch_log_*",
```